core/cli/src/rc-file.ts

```diff
@@ -44,6 +46,27 @@ export async function loadToolKitRC(logger: Logger, root: string, isAppRoot: boo
   }
 
   const config: RawRCFile = result.config
+
+  // if a toolkitrc contains a non-empty options field, but not options.{plugins,tasks,hooks},
```

Comment:
**nitpick**: same as [#615](https://github.com/Financial-Times/dotcom-tool-kit/pull/615#discussion_r1559157732), could you add a TODO or HACK comment here so that it's easier to remember to delete this in a future major version?